### ISSUE TITLE
Frontend dev server starting twice when using option -t

### ISSUE THREAD

**Issue by Menes1337** [shopgate/cloud-sdk#88]
I tried starting the frontend part of my application and the sdk tries to start the frontend twice:

```
> sgconnect frontend start -t weather-station
2018-1-29 09:12:34  INFO: Generating theme config
2018-1-29 09:12:36  INFO: Updated theme config
2018-1-29 09:12:36  INFO: Generating theme config
2018-1-29 09:12:36  INFO: Updated theme config


---------------------------------------------------------------------------

  S H O P G A T E   C L O U D
  D E V E L O P M E N T   S E R V E R

---------------------------------------------------------------------------


  RAPID available. Real data pipeline requests are possible ...


---------------------------------------------------------------------------

  S H O P G A T E   C L O U D
  D E V E L O P M E N T   S E R V E R

---------------------------------------------------------------------------


  RAPID available. Real data pipeline requests are possible ...

  Indexing widgets ...

  Indexing trackers ...


  Indexing widgets ...

  Indexing trackers ...

  Starting Webpack Development Server ...

    STACK: https://sgxs-rapid2-dev.shopgate.services
    SHOP:  31142
  RAPID connecting established!

    STACK: https://sgxs-rapid2-dev.shopgate.services
    SHOP:  31142
  RAPID connecting established!

  The development server is started. You can access the output by:

     http://[..]:8080

  Press CTRL-C to stop the server!

---------------------------------------------------------------------------

events.js:183
      throw er; // Unhandled 'error' event
      ^

Error: listen EADDRINUSE :::9666
    at Object._errnoException (util.js:1024:11)
    at _exceptionWithHostPort (util.js:1046:20)
    at Server.setupListenHandle [as _listen2] (net.js:1351:14)
    at listenInCluster (net.js:1392:12)
    at Server.listen (net.js:1476:7)
    at Server.listen ([..]node_modules/@shopgate/platform-sdk/node_modules/restify/lib/server.js:289:31)
    at setRoutes ([..]node_modules/@shopgate/platform-sdk/lib/app/frontend/rapidDevServer/RapidDevServer.js:64:19)
    at rapidApi.init ([..]/node_modules/@shopgate/platform-sdk/lib/app/frontend/rapidDevServer/RapidDevServer.js:150:9)
    at requestIds ([..]/node_modules/@shopgate/platform-sdk/lib/app/frontend/rapidDevServer/RapidApi.js:38:7)
    at Request.request.post [as _callback] ([..]/@shopgate/platform-sdk/lib/app/frontend/rapidDevServer/RapidApi.js:105:14)
  building [==                  ] [building modules] 10% (0.0 seconds)  Starting Webpack Development Server ...

  building [==                  ] [building modules] 10% (0.0 seconds)events.js:183
      throw er; // Unhandled 'error' event
      ^

Error: listen EADDRINUSE 10.0.77.209:8080
    at Object._errnoException (util.js:1024:11)
    at _exceptionWithHostPort (util.js:1046:20)
    at Server.setupListenHandle [as _listen2] (net.js:1351:14)
    at listenInCluster (net.js:1392:12)
    at doListen (net.js:1501:7)
    at _combinedTickCallback (internal/process/next_tick.js:141:11)
    at process._tickCallback (internal/process/next_tick.js:180:9)
```

**Comment by szll** [comment]
We're on it with: #89

**Comment by pvomhoff** [comment]
Fixed with version v1.2.2
